react-client/styles/style.scss

```diff
@@ -189,6 +191,13 @@ $DialogDetailPadding: 10px 0px 0px 0px;
   }
 }
 
+
+.test-dom {
```

Comment:
What is this???

Reply:
sorry left over during initial testing will remove immediately